madara-runner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+export RPC_API_KEY=$(cat $RPC_API_KEY_FILE)
+
+./madara                   \
```

Comment:
```suggestion
exec tini -- ./madara                   \
``
using exec tini ensures proper signal handling and process lifecycle management within the container

Reply:
using exec tini ensures proper signal handling and process lifecycle management within the container